### PR TITLE
Add tests for camera module #234

### PR DESCRIPTION
**Pull request by stephenjgray**

### CODE REVIEW

**Comment by hirako2000** [comment]
ok with those changes @sasha240100 ?

File: src/modules/app/__tests__/CameraModule.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const aspect = 50;
+  const near = 60;
+  const far = 70;
+
+  const params = {
+    position: position,
+    fov: fov,
+    aspect: aspect,
+    near: near,
+    far: far
```
Comment (sasha240100) [inline src/modules/app/__tests__/CameraModule.spec.js]: Prefer shorthand:

```javascript
const params = {
    position,
    fov,
    aspect,
    near,
    far
}
```

File: src/modules/app/__tests__/CameraModule.spec.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  test('passes a custom aspect value', () => {
+    expect(customCameraModule.camera.native.aspect).toBeCloseTo(aspect, 5);
+  });
+
+  test('passes a custom near value', () => {
+    expect(customCameraModule.camera.native.near).toBeCloseTo(near, 5);
+  });
+
+  test('passes a custom far value', () => {
+    expect(customCameraModule.camera.native.far).toBeCloseTo(far, 5);
```
Comment (sasha240100) [inline src/modules/app/__tests__/CameraModule.spec.js]: Make `customCameraModule.camera.native` to be a variable. It is used frequently

Comment (stephenjgray) [inline src/modules/app/__tests__/CameraModule.spec.js]: Done.

**Pull request merged**